README.md

```diff
@@ -4,73 +4,104 @@
-
-If you are looking for the codebase of the current production version of GATK, please see either the [GATK development framework repository](https://github.com/broadgsa/gatk/) or the [full GATK tools repository](https://github.com/broadgsa/gatk-protected/).
-
-## Requirements
+[Apache Spark](http://spark.apache.org/). It also contains many newly developed tools not present in earlier
+releases of the toolkit.
+
+## Table of Contents
+* [Requirements](#requirements)
+* [Quick Start Guide](#quickstart)
+* [Downloading GATK4](#downloading)
+* [Building GATK4](#building)
+* [Running GATK4](#running)
+    * [Passing JVM options to gatk-launch](#jvmoptions)
+    * [Running GATK4 with inputs on Google Cloud Storage](#gcs)
+    * [Running GATK4 Spark tools on a Spark cluster](#sparkcluster)
+    * [Running GATK4 Spark tools on Google Cloud Dataproc](#dataproc)
+    * [Note on 2bit Reference](#2bit)
+    * [Using R to generate plots](#R)
+    * [Running the CNV workflows](#cnv_workflows)
+* [For GATK Developers](#developers)
+    * [General guidelines for GATK4 developers](#dev_guidelines)
+    * [Testing GATK4](#testing)
+    * [Using Git LFS to download and track large test data](#lfs)
+    * [Creating a GATK project in the IntelliJ IDE](#intellij)
+    * [Setting up debugging in IntelliJ](#debugging)
+    * [Updating the Intellij project when dependencies change](#intellij_gradle_refresh)
+    * [Setting up profiling using JProfiler](#jprofiler)
+    * [Uploading Archives to Sonatype](#sonatype)
+    * [Building GATK4 Docker images](#docker_building)
+    * [Generating GATK4 documentation](#gatkdocs)
+    * [Using Zenhub to track github issues](#zenhub)
+* [Further Reading on Spark](#spark_further_reading)
+* [How to contribute to GATK](#contribute)
+* [Discussions](#discussions)
+* [Authors](#authors)
+* [License](#license)
+
+## <a name="requirements">Requirements</a>
 * Java 8
 * Git 2.5 or greater
 * Optional, but recommended:
-    * Gradle 2.12 or greater, needed for building the GATK. We recommend using the `./gradlew` script which will
+    * Gradle 3.1 or greater, needed for building the GATK. We recommend using the `./gradlew` script which will
       download and use an appropriate gradle version automatically (see examples below).
     * Python 2.6 or greater (needed for running the `gatk-launch` frontend script)
     * R 3.1.3 (needed for producing plots in certain tools, and for running the test suite)
     * [git-lfs](https://git-lfs.github.com/) 1.1.0 or greater (needed to download large files for the complete test suite).
-      Run `git lfs install` after downloading, followed by `git lfs pull` to download the large files. The download is ~500 MB.
+      Run `git lfs install` after downloading, followed by `git lfs pull` from the root of your git clone to download the large files. The download is several hundred megabytes.
```

Comment:
It already says this: `Run `git lfs install` after downloading, followed by `git lfs pull` from the root of your git clone ...`